synapse/tests/test_lib_storm.py

```diff
@@ -3796,6 +3796,27 @@ async def test_storm_argv_parser(self):
         opts = pars.parse_args(['faz', '--cat', 'cam', 'cool'])
         self.nn(opts)
 
+        pars = s_storm.Parser()
+        pars.add_argument('--baz', nargs=3, help='''
+             This is the top line, nothing special.
+             This is my second line with sublines that should have some leading spaces:
+                subline 1: this is a line which has three spaces.
+                  subline 2: this is another line with five leading spaces.
+               subline 3: yet another line with only two leading spaces.
+              subline 4: this line has one space and is long which should wrap around because it exceeds the default display width.
+             This is the final line with no leading spaces.''')
+        pars.add_argument('--taz', type='bool', default=True, help='Taz option')
+        pars.help()
+        self.eq('  --baz <baz>                 : This is the top line, nothing special.', pars.mesgs[5])
+        self.eq('                                This is my second line with sublines that should have some leading spaces:', pars.mesgs[6])
+        self.eq('                                   subline 1: this is a line which has three spaces.', pars.mesgs[7])
+        self.eq('                                     subline 2: this is another line with five leading spaces.', pars.mesgs[8])
+        self.eq('                                  subline 3: yet another line with only two leading spaces.', pars.mesgs[9])
+        self.eq('                                 subline 4: this line has one space and is long which should wrap around because it', pars.mesgs[10])
+        self.eq('                                 exceeds the default display width.', pars.mesgs[11])
+        self.eq('                                This is the final line with no leading spaces.', pars.mesgs[12])
+        self.eq('  --taz <taz>                 : Taz option (default: True)', pars.mesgs[13])
```

Comment:
Follow up on a previous comment, it would be nice if this was dynamically aligned based on the length of the arg and it's placeholder.
```suggestion
        self.eq('  --baz <baz> : This is the top line, nothing special.', pars.mesgs[5])
        self.eq('                This is my second line with sublines that should have some leading spaces:', pars.mesgs[6])
        self.eq('                   subline 1: this is a line which has three spaces.', pars.mesgs[7])
        self.eq('                     subline 2: this is another line with five leading spaces.', pars.mesgs[8])
        self.eq('                  subline 3: yet another line with only two leading spaces.', pars.mesgs[9])
        self.eq('                 subline 4: this line has one space and is long which should wrap around because it', pars.mesgs[10])
        self.eq('                 exceeds the default display width.', pars.mesgs[11])
        self.eq('                This is the final line with no leading spaces.', pars.mesgs[12])
        self.eq('  --taz <taz> : Taz option (default: True)', pars.mesgs[13])
```

Reply:
Not sure I follow what you're asking or suggestion here.

Reply:
I left the base width as-is.